### ISSUE TITLE
Falied to load VLC due to search in wrong locations

### ISSUE THREAD

**Issue by 5UP3RTH30B4G** [WaterMediaTeam/watermedia#63]
I got this error in the console 
Failed to create raw player because VLC is not loaded
and the video menu is black and can't load
![image](https://github.com/SrRapero720/FancyMenu-Video/assets/68150571/630b5942-f4b8-45ed-bd5f-616f696db1f6)

**Comment by SrRapero720** [comment]
need to install VLC for these situations (on linux)
windows and mac have VLC binaries pre-installed

**Comment by 5UP3RTH30B4G** [comment]
i am on Windows

**Comment by SrRapero720** [comment]
send me logs via github gist

**Comment by 5UP3RTH30B4G** [comment]
[latest.log](https://github.com/SrRapero720/FancyMenu-Video/files/15077752/latest.log)
https://gist.github.com/Supertheobrg/8b07cc8d4bc6047aed19b5842fbd0ce0
The error is located at Line 1279

**Comment by SrRapero720** [comment]
those errors had NOTHING related to watermedia or fmv. your problem is WATERMeDIA is attemping to load VLC but cannot be founded

go to ``C:\Users\theob\AppData\Local\Temp\watermedia\`` and delete the entire folder
if that doesn't work install VLC 3 manually

![image](https://github.com/SrRapero720/FancyMenu-Video/assets/29090346/3d938441-5a24-49ae-a458-0832baedc9d4)

**Comment by 5UP3RTH30B4G** [comment]
No I still have the Issue...

**Comment by SrRapero720** [comment]
can you open VLC normally? with no issues?

**Comment by 5UP3RTH30B4G** [comment]
yup VLC work fine

**Comment by 5UP3RTH30B4G** [comment]
The mod doesn't search at the right place, the mod is not looking in the installation folders by default :
He's located by default here : C:\Program Files (x86)\VideoLAN to fix this issue i had to copy the folders into one of those location

```
Searching on 'C:\Users\theob\AppData\Local\Temp\watermedia\videolan
C:\Users\theob\curseforge\minecraft\Instances\FancyMenu
C:\Program Files (x86)\Razer Chroma SDK\bin
C:\Program Files\Razer Chroma SDK\bin
C:\Program Files (x86)\VMware\VMware Player\bin\
C:\Program Files\Eclipse Adoptium\jre-8.0.402.6-hotspot\bin
C:\Program Files\Eclipse Adoptium\jre-17.0.10.7-hotspot\bin
C:\Program Files\Eclipse Adoptium\jre-11.0.22.7-hotspot\bin
C:\Program Files (x86)\Eclipse Adoptium\jre-8.0.402.6-hotspot\bin
C:\ProgramData\Oracle\Java\javapath
C:\Program Files\Common Files\Oracle\Java\javapath
C:\Program Files (x86)\Common Files\Oracle\Java\java8path
C:\Program Files (x86)\Common Files\Oracle\Java\javapath
C:\Windows\system32
C:\Windows\
C:\Windows\System32\Wbem
C:\Windows\System32\WindowsPowerShell\v1.0\
C:\Windows\System32\OpenSSH\
C:\Program Files\PuTTY\
C:\Program Files\Microsoft VS Code\bin
C:\Program Files (x86)\dotnet\
C:\Program Files\dotnet\
C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common
C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR
C:\Windows\system32\config\systemprofile\AppData\Local\Microsoft\WindowsApps
C:\Program Files (x86)\Razer\ChromaBroadcast\bin
C:\Program Files\Razer\ChromaBroadcast\bin
C:\Users\theob\AppData\Local\Microsoft\WindowsApps
```

For me, the one that worked, is **C:\Windows\**

**Comment by 5UP3RTH30B4G** [comment]
Copy your VideoLAN folders into one of this location
```
C:\Users\theob\curseforge\minecraft\Instances\FancyMenu
C:\Program Files (x86)\Razer Chroma SDK\bin
C:\Program Files\Razer Chroma SDK\bin
C:\Program Files (x86)\VMware\VMware Player\bin\
C:\Program Files\Eclipse Adoptium\jre-8.0.402.6-hotspot\bin
C:\Program Files\Eclipse Adoptium\jre-17.0.10.7-hotspot\bin
C:\Program Files\Eclipse Adoptium\jre-11.0.22.7-hotspot\bin
C:\Program Files (x86)\Eclipse Adoptium\jre-8.0.402.6-hotspot\bin
C:\ProgramData\Oracle\Java\javapath
C:\Program Files\Common Files\Oracle\Java\javapath
C:\Program Files (x86)\Common Files\Oracle\Java\java8path
C:\Program Files (x86)\Common Files\Oracle\Java\javapath
C:\Windows\system32
C:\Windows\
C:\Windows\System32\Wbem
C:\Windows\System32\WindowsPowerShell\v1.0\
C:\Windows\System32\OpenSSH\
C:\Program Files\PuTTY\
C:\Program Files\Microsoft VS Code\bin
C:\Program Files (x86)\dotnet\
C:\Program Files\dotnet\
C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common
C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR
C:\Windows\system32\config\systemprofile\AppData\Local\Microsoft\WindowsApps
C:\Program Files (x86)\Razer\ChromaBroadcast\bin
C:\Program Files\Razer\ChromaBroadcast\bin
C:\Users\theob\AppData\Local\Microsoft\WindowsApps```

**Comment by SrRapero720** [comment]
i transfered the issue because is a unknow issue from watermedia
VLCJ gets the wrong path to the VLC installation for some unknw reason (blaming broken registries)

**Comment by SrRapero720** [comment]
@Supertheobrg i searched into discovery stuff and indeed. discovery uses the windows registry installation path

can you show me what is the VLC installation path in your registry editor?
inside on `Equipo\HKEY_LOCAL_MACHINE\SOFTWARE\VideoLAN\VLC`

should look like this:
![image](https://github.com/SrRapero720/watermedia/assets/29090346/c50befb2-b49d-41be-99f1-c92a56ff9bf4)

**Comment by 5UP3RTH30B4G** [comment]
![image](https://github.com/SrRapero720/watermedia/assets/68150571/55bc5a5d-b591-476f-8cc5-343c8c312e5c)
I don't see it

**Comment by SrRapero720** [comment]
try reinstalling VLC... aparently other people fixes it  reinstalling VLC

**Comment by SrRapero720** [comment]
issue was fixed in 2.0.41. and enhanced in 2.0.42
